documentation/release-notes-2-8.md

```diff
@@ -34,27 +34,39 @@ and finally a boot loader is installed.
 System administrators use the ReaR framework to set up a disaster recovery procedure
 as part of their disaster recovery policy (which complements their existing backup policy).
 
-ReaR complements backup and restore of data
-but ReaR is neither a backup software nor a backup management software.
-Data backup and restore happens via dedicated backup software which is called by ReaR.
-
-ReaR has built-in support for some 'internal' backup programs like 'tar' and 'rsync'.
-
-ReaR integrates with various external (third-party) backup software solutions, for example:
-
-* IBM Storage Protect (TSM) / Tivoli Storage Manager / IBM Spectrum Protect
-* Data Protector (DP)
-* Veritas NetBackup (NBU) / Symantec NetBackup
-* Commvault Simpana (GALAXY10) / Galaxy 10
-* Bacula (BACULA)
-* Bareos (BAREOS)
-* Rsync Backup Made Easy (RBME)
-* Duplicity/Duply (DUPLICITY)
-* Dell NetWorker (NSR) / EMC NetWorker / Legato NetWorker
-* Dell EMC Avamar (AVA) / EMC Avamar
-* SEP Sesam (SESAM)
-* Borg (BORG)
-* Rubrik Cloud Data Management (CDM)
+ReaR complements backup and restore of data with bare metal disaster recovery. ReaR can also act as local backup software,
+but ReaR is not a a backup management software. In many enterprise environmentments, data backup and restore happens via dedicated backup software which is integrated by ReaR and used to restore the data onto a recovery system.
+
+ReaR has support for built-in backup methods using 'tar' and 'rsync' that are used for backup and restore.
+
+ReaR integrates supports the following 3rd party, also commercial, tools for restoring a backup.
+
+The complete list of backup methods (`BACKUP=...`) is:
+
+* `AVA` Dell EMC Avamar / EMC Avamar
+* `BACULA` Bacula
+* `BAREOS` Bareos
+* `BLOCKCLONE` block device cloning via `dd`
+* `BORG` Borg Backup
+* `CDM` Rubrik Cloud Data Management
+* `DP` HP Data Protector
```

Comment:
HP does not own DP anymore - see https://www.opentext.com/products/data-protector